### PR TITLE
fix: open collapsible on schema refresh

### LINKED ISSUE #25414: [Feature]: If the schema section is collapsed, clicking on refresh should expand the the schema section
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

source: https://docs.google.com/spreadsheets/d/1QOXj3w4QuFkG9N7fwfYsOgErrXZn5d9VZleO3XId8zQ/edit#gid=0&range=6:6

notion: https://www.notion.so/appsmith/Activation-60c64894f42d4cdcb92220c1dbc73802?p=401ba6eeb92542acb011f0109eacc2f9&pm=s

https://github.com/appsmithorg/appsmith/assets/94514895/16cc9c73-28d7-4e38-a838-4cb757c6c7e0

### Why should this be worked on?

activation project

### PR DESCRIPTION
**Pull request by Irongade**
This PR ensures the collapsible is opened when the schema refresh button is clicked.

Fixes #25414 

- Bug fix (non-breaking change which fixes an issue)

- [x] Manual
- [ ] Jest
- [x] Cypress
>
>
#### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [x] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [x] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

File: app/client/src/pages/Editor/Explorer/Libraries/recommendedLibraries.ts [hunk 1]
```diff
@@ -47,6 +47,15 @@ export default [
     version: "4.2.12",
     icon: "https://github.com/LiosK.png?s=20",
   },
+  {
```
Comment (sneha122) [inline app/client/src/pages/Editor/Explorer/Libraries/recommendedLibraries.ts:50]: @Irongade Why are we adding this library object here?

Comment (Irongade) [reply app/client/src/pages/Editor/Explorer/Libraries/recommendedLibraries.ts:50]: This was probably added after I merged release into my branch. It is a part of @ohansFavour's changes.

Comment (sneha122) [reply app/client/src/pages/Editor/Explorer/Libraries/recommendedLibraries.ts:50]: But even if you merge release in your branch, you should not see those changes in your PR. If we want these changes then we can keep it.

**Review by sneha122 — DISMISSED** [review]

**Review by sneha122 — APPROVED** [review]

**Pull request merged**